src/GT.UI/Program.cs

```diff
@@ -52,8 +52,8 @@
 	.AddTransient<IGTIdentityRepository, GTIdentityRepository>();
 
 // Add BLL services
-builder.Services
-	.AddTransient<IGTListingService, GTListingService>();
+//builder.Services
+//	.AddTransient<IGTListingService, GTListingService>();
 
```

Comment:
Need confirmation that these are not still commented out in main once PR is merged.